config/database.php

```diff
@@ -68,10 +68,10 @@
 
         'pgsql' => [
             'driver'   => 'pgsql',
-            'host'     => env('DB_HOST', $url["host"]),
-            'database' => env('DB_DATABASE', substr($url["path"], 1)),
-            'username' => env('DB_USERNAME', $url["user"]),
-            'password' => env('DB_PASSWORD', $url["pass"]),
+            'host'     => env('DB_HOST', localhost),
+            'database' => env('DB_DATABASE', 'thehub'),
+            'username' => env('DB_USERNAME', 'postgres'),
+            'password' => env('DB_PASSWORD', 'Olanicmi2013'),
```

Comment:
i need you to revert this commit. You are exposing your credentials to the public. To revert, you can check for more infomation on that [here](http://goodheads.io/2015/08/18/think-like-a-git-commit-like-a-boss-revert-now/)